code/lib/core-server/src/utils/StoryIndexGenerator.test.ts

```diff
@@ -670,7 +652,6 @@ describe('StoryIndexGenerator', () => {
                 "standalone": true,
                 "storiesImports": Array [],
                 "tags": Array [
-                  "mdx",
```

Comment:
Does `docs-mdx` need to be updated to produce the `mdx` tag?

Reply:
I guess so. I think we're still a bit unsure about what exactly the `mdx` tag means, but if it means "this entry comes from a mdx file" then, yes, it should have that tag.

Reply:
Added it to the StoryIndexGenerator. Unclear what should go there vs in the `docs-mdx` analyzer